packages/block-library/src/navigation-menu/index.php

```diff
@@ -97,31 +97,36 @@ function build_navigation_menu_html( $block, $colors ) {
 	$html = '';
 	foreach ( (array) $block['innerBlocks'] as $key => $menu_item ) {
 
-		$html .= '<li class="wp-block-navigation-menu-item ' . $colors['bg_css_classes'] . '"' . $colors['bg_inline_styles'] . '>' .
-			'<a
-				class="wp-block-navigation-menu-item__link ' . $colors['text_css_classes'] . '"
-				' . $colors['text_inline_styles'];
-
-		// Start appending HTML attributes to anchor tag.
+		// Creates the markup for the item content element.
 		if ( isset( $menu_item['attrs']['url'] ) ) {
-			$html .= ' href="' . $menu_item['attrs']['url'] . '"';
-		}
-		if ( isset( $menu_item['attrs']['title'] ) ) {
-			$html .= ' title="' . $menu_item['attrs']['title'] . '"';
+			$item_content_markup =
+				'<a
+					class="wp-block-navigation-menu-item__link ' . $colors['text_css_classes'] . '"
+					' . $colors['text_inline_styles'] .
+					' href="' . $menu_item['attrs']['url'] .
+					( $menu_item['attrs']['title'] ? ( ' title="' . $menu_item['attrs']['title'] . '"' ) : '' ) .
+					( isset( $menu_item['attrs']['opensInNewTab'] ) && true === $menu_item['attrs']['opensInNewTab']
+						? ' target="_blank"'
+						: ''
+					) .
+				'>' .
+					( isset( $menu_item['attrs']['label'] ) ? $menu_item['attrs']['label'] : '' ) .
+				'</a>';
+		} else {
+			$item_content_markup =
+				'<label
```

Comment:
Any reason for this to be a `<label>`? Labels with no form fields associated have no semantic value, so you might as well use a `<span>` and avoid having to override label-specific styles. I'm also thinking this could become weirdly unsemantic if we ever decide to add other types of markup inside the nav items.

Reply:
+1 - a span is better than an empty label

Reply:
👍 I changed for a `<label>` in the last commit. Let me rever it.